cli/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "reinfer-cli"
 version = "0.11.0"
-description = "Command line interface for the reinfer platform."
+description = "Command line interface for the Re:infer conversational data intelligence platform"
```

Comment:
Again, weird phrasing, what are you trying to say :sweat_smile:, what's "the Re:infer"?

Reply:
Hahaha, I just copy pasta'ed from somewhere else, apparently with no regard for grammer :sweat_smile: